AI/app/data/run/run_chroma.py

```
@@ -0,0 +1,41 @@
+        a.city_code
+    FROM attraction a
+    JOIN attraction_category ac 
+        ON a.attraction_category_id = ac.attraction_category_id
+"""
+
+CHROMA_DB_PATH = os.path.join(PROJECT_ROOT, "data/chroma_db")
+
+def main():
+    rows = execute_query(QUERY)
+    documents = convert_to_documents(rows)
+    vector_store = save_to_chroma(documents, CHROMA_DB_PATH)
```

Ruff (0.11.9)

Local variable `vector_store` is assigned to but never used

Remove assignment to unused variable `vector_store`

(F841)